### PR TITLE
feat: review app copy

### LINKED ISSUE #225: Station UI Copy review
Review app copy. Check [notion page](https://www.notion.so/subvisual/Desktop-app-content-1545b042618a464ea9a91189fdf870c7)

### PR DESCRIPTION
**Pull request by jopedroliveira**
Closes #225
This PR adds the first reviewed version of copy and some tweaks on CSS to better fit the copy in the containers.

### CODE REVIEW

**Comment by juliangruber** [comment]
Screenshot 2022-10-13 at 12 40 40

This screen looks out of balance to me now 🤔

**Comment by jopedroliveira** [comment]
> Screenshot 2022-10-13 at 12 40 40
> 
> This screen looks out of balance to me now 🤔

I agree. It's a draft, I'm trying to sync with a designer for this

**Review by juliangruber — CHANGES_REQUESTED** [review]
Aside from the loading screen, Lgtm! :)

**Comment by jopedroliveira** [comment]
> Aside from the loading screen, Lgtm! :)

João Ferreira gave a good idea:
Change the step 2 Title to “Join the Filecoin Economy” but keep the text. And remove the slogan from splash screen

File: renderer/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const Dashboard = (): JSX.Element => {
               <p className="w-fit text-header-m font-bold font-number total-jobs" title="total jobs">{totalJobs}</p>
             </div>
             <div className="mb-6">
-              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Total Earnings (updated daily)</p>
+              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Total Earnings (comming soon)</p>
```
Comment (bajtos) [inline renderer/src/pages/Dashboard.tsx:81]: I think the correct spelling is `coming`?

```suggestion
              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Total Earnings (coming soon)</p>
```

**Review by juliangruber — APPROVED** [review]

**Pull request merged**